foodsaving/groups/receivers.py

```diff
@@ -29,9 +30,13 @@ def group_member_added(sender, instance, **kwargs):
     """When a user is removed from a conversation we will notify them."""
     group = instance.group
     user = instance.user
+    roles = instance.roles
     conversation = Conversation.objects.get_or_create_for_target(group)
-    conversation.join(user)
-    stats.group_joined(group)
+    if GROUP_APPROVED_MEMBER in roles:
+        conversation.join(user)
+        stats.group_joined(group)
```

Comment:
This will be called whenever an approved group membership is saved (e.g. all the lastseen_at times) so our stats will be wildly optimistic :)

Reply:
... although looking at master, this is already the case